### PR TITLE
implemented single page browsing

### PR DESCRIPTION
**Pull request by wolkemann**

### CODE REVIEW

File: components/OuterWindow/OuterWindow.js [hunk 1]
```diff
@@ -11,8 +11,8 @@ const OuterWindow = styled.div`
   margin: 0.5rem 0;
   padding: 3px;
   color: var(--text-color);
-  border: 2px solid var(--window-innerborder-color);
+  border: 2px solid var(--window-border-color);
```
Comment (mbosselmann) [inline components/OuterWindow/OuterWindow.js:14]: I like that you work with CSS variables! 👍

**Review by mbosselmann — APPROVED** [review]
Nice work! 🚀 Keep it up! 👍 I only have two ideas you can think about. :)

File: components/OuterWindow/OuterWindow.js (hunk 1 above)
Comment (wolkemann) [reply components/OuterWindow/OuterWindow.js:14]: <3

File: components/Letter/Letter.js [hunk 2]
```diff
@@ -1,21 +1,35 @@
...
 import useSWR from "swr";
 import Link from "next/link";
 
 import { Icon } from "@iconify/react";
 import { Button } from "../Button/Button";
 
-export default function Letter({ children, authorId, replyId, showActions }) {
+export default function Letter({
+  children,
+  isAReply,
```
[the remark this answers is not in the document]
Comment (wolkemann) [reply components/Letter/Letter.js:11]: Thank you for your suggestion? IsAReply would mean that this Letter is a "reply" letter and so the background color should change. Maybe a more clear name could be `isAReplyLetter`?

File: pages/inbox/browse.js [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+export async function getServerSideProps(context) {
+  const session = await getSession(context);
+  if (!session) {
+    return {
+      redirect: {
+        destination: "/signin/",
+        permanent: false,
+      },
+    };
+  }
```
[the remark this answers is not in the document]
Comment (wolkemann) [reply pages/inbox/browse.js:47]: For sure! This is my plan after I finish the core functions of the App!

File: components/Letter/Letter.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (djfarly) [reply components/Letter/Letter.js:11]: maybe simply `isReply` - i think it the `A` that is throwing me off for some reason

[the remark this answers is not in the document]
Comment (mbosselmann) [reply components/Letter/Letter.js:11]: Ah, now I did understand it correctly! 👍 `isAReplyLetter` seems to me better to understand then `isAReply`, but I think you can also work with `isReplyLetter`. :)

[the remark this answers is not in the document]
Comment (wolkemann) [reply components/Letter/Letter.js:11]: Super! thanks both @mbosselmann and @djfarly

**Pull request merged**